test/javascripts/lib/load-script-test.js.es6

```diff
@@ -0,0 +1,23 @@
+import loadScript from "discourse/lib/load-script";
+
+QUnit.module("lib:load-script");
+
+QUnit.test(
+  "load with a script tag, and callbacks are only executed after script is loaded",
+  async assert => {
+    const src = "/javascripts/ace/ace.js";
+
+    await loadScript(src).then(() => {
+      assert.ok(
+        typeof ace !== "undefined",
+        "callbacks should only be executed after the script has fully loaded"
+      );
+
+      const scriptTags = Array.from(document.getElementsByTagName("script"));
```

Comment:
I imagine this can't use the ember test helpers because the script tag is injected outside of the test sandbox frame?

Reply:
(probably should add a comment if so - otherwise someone might copy and paste this test and not be searching the dom with the correct restrictions.)